### PR TITLE
Updated the GATK Example

### PR DESCRIPTION
**Pull request by Nick-Harvey**
changed the location of the GATK data file and made some minor updates to the copy.

### CODE REVIEW

File: doc/examples/gatk/README.md [hunk 1]
```diff
@@ -41,21 +55,23 @@ refSDF              dir                 22.57MiB
...
+Next, we're going to work on commiting the `*.bam` files for Mom. Let's stary by create a repositories for input `*.bam` files to go into:
 
 ```sh
 $ pachctl create-repo samples
 ```
 
-Add a `*.bam` file (along with it's index file) corresponding to a first sample (`mother`). Here we will assume that the files corresponding to each sample are committed to separate directories (e.g., `/mother`):
+Add a `*.bam` file (along with it's index file) corresponding to a first sample, in our case it's the `mother`. Here we will assume that the files corresponding to each sample are committed to separate directories (e.g., `/mother`). 
+
+**Important**: make sure to replace where it says `<yourcommitid>` with the commit-id created by `pachctl start-commit`
```
Comment (JoeyZwicker) [inline doc/examples/gatk/README.md]: Do we need to use commitIDs for some reason here or can we just use branch name `master`

File: doc/examples/gatk/README.md [hunk 2]
```diff
@@ -8,8 +8,11 @@ This example illustrates the use of GATK in Pachyderm for [Germline](https://en.
 
 You can retrieve all the input data for this pipeline from the Broad Institute [here](https://drive.google.com/open?id=0BzI1CyccGsZicE5HNkR6anpLTnM). We will utilize a b37 human genome reference containing only a subset of chromosome 20, which we prepared specially for GATK tutorials in order to provide a reasonable size for download. It is accompanied by its index and sequence dictionary.
```
Comment (JoeyZwicker) [inline doc/examples/gatk/README.md]: Do you want to set some context for what GATK is and why is useful?

File: doc/examples/gatk/README.md [hunk 3]
```diff
@@ -22,14 +25,25 @@ Archive:  GATK_Germline.zip
...
+First milestone reached! Lets just check and make sure everything looks good
+```sh
 $ pachctl list-repo
 NAME                CREATED             SIZE
 reference           43 seconds ago      83.68MiB
+```
+```sh
 $ pachctl list-file reference master
 NAME                TYPE                SIZE
 DS_Store            file                8.004KiB
```
Comment (JoeyZwicker) [inline doc/examples/gatk/README.md]: I dont think this DS store file should be there. if I remember correctly, DS_store is some weird mac figment that it seems like got pushed into the repo along with the data. I dont think it'll break anything. Or maybe I'm totally wrong and that's a real part of the demo

**Comment by JoeyZwicker** [comment]
@Nick-Harvey This is still waiting for the changes made to be merged.

**Comment by Nick-Harvey** [comment]
closing this PR and opening a separate one with the requested changes.

**Pull request closed without merging**